handbook/product/planning.md

```diff
@@ -43,8 +43,10 @@ Tracking issues are GitHub issues that encapsulate the planned work for an itera
 
 Teams should aggressively prioritize work for the iteration so that expectations can be set accordingly with users waiting for features or bug fixes. If work is carried over for multiple iterations it is an indication that the priority of that item is not actually as high as believed, or that the team is trying to do too many things in an iteration. Engineering can lean on Product for help with prioritization.
 
-## Prioritizing bug fixes and small tasks
+### Delivery plans
```

Comment:
```suggestion
### [Delivery plans](delivery_plans.md)
```

This makes it so the link in this page's table of contents goes directly to the delivery_plans page instead of to the section (which contains only a link).